src/ws/index.js

```diff
@@ -1,19 +1,36 @@
-let sandbox = require('./send-sandbox')
-let run = require('./send')
+let { ApiGatewayManagementApi } = require('aws-sdk')
+let http = require('http')
+let { sandboxVersionAtLeast } = require('../sandbox')
+
+let ARC_WSS_URL = process.env.ARC_WSS_URL
+let port = process.env.ARC_INTERNAL || '3332'
+let local = process.env.NODE_ENV === 'testing' || process.env.ARC_LOCAL
+
+let apiGatewayManagementApi
```

Comment:
Aesthetic, I know, but per [how we do it for DynamoDB](https://arc.codes/docs/en/reference/runtime-helpers/node.js#arc.tables), can we rename this from `apiGatewayManagementApi` to `_api`?